pkg/skaffold/build/local/types.go

```diff
@@ -90,26 +95,23 @@ func NewBuilder(cfg Config, buildCfg *latest.LocalBuild) (*Builder, error) {
 
 	return &Builder{
 		local:              *buildCfg,
-		cfg:                cfg,
-		kubeContext:        cfg.GetKubeContext(),
+		cfg:                bCtx,
```

Comment:
It seems odd to be pulling out the different values from the config/context (kubecontext, skiptests, mode, etc) when we could just retain the config/context object.  I suppose ultimately this config object is the RunContext, which has mutable fields.